tests/framework/helpers/FileHelperTest.php

```diff
@@ -4,7 +4,6 @@
  * @copyright Copyright (c) 2008 Yii Software LLC
  * @license http://www.yiiframework.com/license/
  */
-
 use yii\helpers\FileHelper;
```

Comment:
Why remove this line?

Reply:
Maybe after updating to the latest version of the `php-cs-fixer`.